packages/mgt-components/src/graph/graph.user.ts

```diff
@@ -254,6 +258,8 @@ export async function getUsersForPeopleQueries(graph: IGraph, peopleQueries: str
         if (getIsUsersCacheEnabled()) {
           cache.putValue(personQuery, { maxResults: 1, results: [JSON.stringify(response.content.value[0])] });
         }
+      } else {
+        if (people.length < peopleQueries.length) people.push(null);
```

Comment:
No need to check the length here.

```suggestion
        people.push(null);
```